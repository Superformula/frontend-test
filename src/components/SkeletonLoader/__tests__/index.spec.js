import React from 'react';
import { mount } from 'enzyme';
import { SkeletonLoader } from '../';

describe('SkeletonLoader', () => {
  it('should render without failing', () => {
    const wrapper = mount(<SkeletonLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
