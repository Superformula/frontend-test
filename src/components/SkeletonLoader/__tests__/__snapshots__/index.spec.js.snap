// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonLoader should render without failing 1`] = `
.c0 {
  display: inline-block;
  line-height: 1;
  width: 100%;
  height: 100%;
  color: transparent;
  background: #D8D8D8;
  background: linear-gradient(90deg,#D8D8D8,#F2F2F2);
  background-size: 400% 400%;
  -webkit-animation: jzvvOC 2s ease infinite;
  animation: jzvvOC 2s ease infinite;
}

<SkeletonLoader>
  <span
    className="c0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "&nbsp;",
      }
    }
    role="progressbar"
  />
</SkeletonLoader>
`;
